playbooks/tasks/portal-execute-commands.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Execute any defined docker commands
+- name: Executing Docker Command
+  include_tasks: tasks/portal-docker-command.yml
+
+# Block any defined hosts
+- name: Block any bad hosts
+  ansible.builtin.command: "{{ item }}"
+  loop: "docker exec sia siac hostdb block {{ item }}"
```

Comment:
`loop` and `command` are not defined correctly, what do we iterate over?

Reply:
whoops, good catch